src/app/find/mapPage/filters/filters.css

```diff
@@ -0,0 +1,69 @@
+@value primaryColor from "../../../../colors.css";
+
+.doneBtnContainer{
+	position: fixed;
+	bottom: 0;
+	width: 100%;
+	z-index: 1040;
+}
+
+.doneBtn{
+	border: 2px solid primaryColor;
+	background-color: #ffffff;
+	min-height: 56px;
+	width: 96%;
+	margin: 2%;
+	border-radius: 4px;
+	text-transform: capitalize;
+	color: primaryColor;
+	letter-spacing: 0px;
+	font-size: 1rem;
+	padding: 0px !important;
+	display: flex;
```

Comment:
What children does the done button even have, that make it useful to use flex here?